homeassistant/components/zwave.py

```diff
@@ -92,6 +93,10 @@
      [COMMAND_CLASS_DOOR_LOCK],
      TYPE_BOOL,
      GENRE_USER),
+    ('rollershutter',
+     [COMMAND_CLASS_SWITCH_MULTILEVEL],
```

Comment:
Ah yes - here the [openhab](https://github.com/openhab/openhab/blob/master/bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/ZWaveDeviceClass.java) class - or the [openzwave](https://github.com/wokar/open-zwave/blob/master/config/device_classes.xml) mapping which better shows the relation of generic, specific and required command classes.